packages/compass-import-export/src/import/analyze-csv-fields.spec.ts

```diff
@@ -198,7 +199,7 @@ describe('analyzeCSVFields', function () {
     });
 
     // only looked at the first row because we aborted before even starting
-    expect(result.totalRows).to.equal(1);
+    expect(result.totalRows).to.equal(0);
```

Comment:
We now abort the pipeline using node stream promises's pipeline machinery so the timing it subtly different. It is kinda arbitrary whether we looked at 0, 1 or 2 rows before it kicked in.